projects/ebv-analyzer-app/src/app/ebv-selector/ebv-selector.component.html

```diff
@@ -122,23 +122,18 @@
 
     <mat-spinner *ngIf="loading$ | async"></mat-spinner>
 
-    <div *ngIf="false">
-        <mat-divider></mat-divider>
-
-        <!-- <div fxLayout="row" fxLayoutAlign="space-between start">
-            <wave-ebv-country-selector fxFlex=""></wave-ebv-country-selector>
-            <span fxFlex="1rem"></span>
-            <button mat-raised-button color="primary" [disabled]="isPlotButtonDisabled$ | async" (click)="plot()">Calculate</button>
-        </div> -->
+    <mat-divider></mat-divider>
 
-        <!-- TODO: implement -->
-        <!-- <wave-ebv-indicator-plot *ngIf="plotSettings"
-                                 [data]="plotSettings.data$"
-                                 [xLimits]="plotSettings.xLimits"
-                                 [yLabel]="plotSettings.yLabel"
-                                 [yLimits]="plotSettings.yLimits"></wave-ebv-indicator-plot> -->
+    <div fxLayout="row" fxLayoutAlign="space-between start">
+        <wave-ebv-country-selector fxFlex=""></wave-ebv-country-selector>
+        <span fxFlex="1rem"></span>
+        <button mat-raised-button color="primary" [disabled]="isPlotButtonDisabled$ | async" (click)="plot()">Calculate</button>
     </div>
 
+    <wave-vega-viewer *ngIf="(plotData | async) !== undefined" [chartData]="plotData | async"></wave-vega-viewer>
```

Comment:
Didn't you have your trick of binding to the output of ngIf and using that in the other field?

Reply:
What trick? I copied this snipped from the plots in the others portals

Reply:
You bounded the async call to a variable and could use the variable then without having to subscribe twice. You introduced this concept…

Reply:
f79b05467731ed2d842d28424e885a8f6cd4dd58